Poker/Deck.cs

```diff
@@ -7,13 +7,24 @@ namespace Poker
 {
     public class Deck
     {
-        public List<Card> listOfCards { get; }
+        public List<Card> ListOfCards { get; }
+
+        public int CurrentCard { get; private set; }
```

Comment:
The purpose of this field is to work hand in hand with the new DrawCardFromDeck. The responsibility of this should belong to the deck, not the game. The game simply asks the deck for a card, and the deck figures out which one to give it. It is a simple index that increases every time a card is drawn